config/environments/test.rb

```diff
@@ -40,8 +42,15 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
Drive by comment: this option and the one underneath ([annotate_rendered_view_with_filenames](https://github.com/alphagov/manuals-publisher/pull/1783/commits/7f26b579d09b0f000310bca985fa963fec3504b2#diff-77f322f5ab0c54b1f2793b339574333dc55889645d910a93ede0fd7aa13b217aR55)) should be enabled for the test and dev environments

Reply:
Thanks @1pretz1 ! I've enabled them now.